redis/src/main/java/org/togglz/redis/RedisStateRepository.java

```diff
@@ -0,0 +1,151 @@
+package org.togglz.redis;
+
+import org.togglz.core.Feature;
+import org.togglz.core.repository.FeatureState;
+import org.togglz.core.repository.StateRepository;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+import java.util.Map;
+
+/**
+ * A state repository which stores the feature state in Redis.
+ * <p>
+ * The class provides a builder which can be used to configure the state repository instance
+ * (e.g. Redis hostname, Redis/Jedis configuration, serialization).
+ * </p>
+ *
+ * @author Cosmin Rentea
+ */
+public class RedisStateRepository implements StateRepository {
+
+    public static final String KEY_PREFIX = "togglz:";
```

Comment:
A minor one. Could you either inline this or move the constant to the builder. I was actually very confused by this. At first my impression was that this is a fixed prefix for the keys, but actually it is just the default prefix used if the user didn't specify another one. Therefore I think it would be ok to either inline it or move it to the build and name it `DEFAULT_KEY_PREFIX`.